enarx-keep-sgx-shim/src/heap.rs

```diff
@@ -138,12 +137,12 @@ impl Heap {
         fd: isize,
         offset: usize,
     ) -> usize {
-        const RWX: u64 = nolibc::PROT_READ | nolibc::PROT_WRITE | nolibc::PROT_EXEC;
-        const PA: u64 = nolibc::MAP_PRIVATE | nolibc::MAP_ANONYMOUS;
+        const RWX: u64 = libc::PROT_READ as u64 | libc::PROT_WRITE as u64 | libc::PROT_EXEC as u64;
+        const PA: u64 = libc::MAP_PRIVATE as u64 | libc::MAP_ANONYMOUS as u64;
```

Comment:
It might be easier to have these values be the same type as the `libc` constants and do a final cast later in the code.